tcms/search/query.py

```diff
@@ -99,7 +99,8 @@ class SmartDjangoQuery(object):
     }
 
     def __init__(self, queries, result_kls):
-        self.queryset = self.CONTENT_TYPES[result_kls]._default_manager.all()
+        self.queryset = self.CONTENT_TYPES[result_kls]\
+            ._default_manager.all()  # pylint: disable=protected-access
```

Comment:
`_default_manager` -> `objects` because for all 3 models in the CONTENT_TYPES dictionary we don't override the default objects manager.